native/metro.config.js

```diff
@@ -2,8 +2,9 @@
 const { getDefaultConfig, mergeConfig } = require('@react-native/metro-config')
 const path = require('path')
 
-/** @type {import('metro-config').MetroConfig} */
+/** @type {import('@react-native/metro-config').MetroConfig} */
 const defaultConfig = getDefaultConfig(__dirname)
+defaultConfig.resolver.unstable_enablePackageExports = true
```

Comment:
This is needed for `react-navigation-header-buttons`, and it will be enabled by default in a future version of `react-native`

https://github.com/vonovak/react-navigation-header-buttons/blob/master/INSTALL.md#installation--setup